whipper/program/cdrdao.py

```diff
@@ -43,8 +43,7 @@ def read_toc(device, fast_toc=False):
 
     toc = TocFile(tocfile)
     toc.parse()
-    os.unlink(tocfile)
```

Comment:
Does this mean that when the tocfile is unused (like for `whipper offset`), we have a `/tmp`-file that isn't cleaned up?